opm/grid/GraphOfGrid.hpp

```diff
@@ -162,9 +162,24 @@ class GraphOfGrid{
         return wells;
     }
 
+    /// \brief Contract a layer of verices around each wells into it
+    ///
+    /// Representing a well by one node guarantees that the well won't
+    /// be split over several processes. Giving the well an extra layer
+    /// of cells distances that well from the subdomain boundary.
+    void addWellBuffer ();
+    void addWellBuffer (int layers)
```

Comment:
It would be nice if the function name would give a better hint about what the functions do.
Please document the other function, too.